salt/config/__init__.py

```diff
@@ -919,6 +919,14 @@ def _gather_buffer_space():
 
     # Minion data cache driver (one of satl.cache.* modules)
     'cache': str,
+    # Enables a fast in-memory cache booster and sets the expiration time.
+    'memcache_expire_seconds': int,
```

Comment:
I was just wondering: how is this going to influence the caching etc? I would be worried if this requires to execute `refresh_pillar` more often, for example.